.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "0.12"
+  - "5.9.1"
```

Comment:
Ok great, I was seeing that there are some npm packages that needed an updated node version, but I didnt know that 4.x is going to be supported till 2018. Ill fix it right away :smile: